pkg/spec/backup.go

```diff
@@ -40,7 +40,7 @@ type BackupPolicy struct {
 	// StorageType specifies the type of storage device to store backup files.
 	// If it's not set by user, the default is "PersistentVolume".
 	StorageType BackupStorageType `json:"storageType"`
-	// CleanupOnClusterDelete specified whether we want to cleanup the backup data if cluster is deleted.
+	// CleanupStorageOnDelete specified whether we want to cleanup the backup storage data if cluster is deleted.
 	// By default, operator will keep the backup data.
-	CleanupOnClusterDelete bool `json:"cleanupOnClusterDelete"`
+	CleanupStorageOnDelete bool `json:"cleanupStorageOnDelete"`
```

Comment:
We need to be specific on what Delete here. this in the backup policy. so on delete of backup (we might support backup policy change in the future)? or on delete of the cluster?